content/en/docs/examples/local-testing-with-kind/index.md

```diff
@@ -0,0 +1,388 @@
+---
+title: Testing Istio Locally With Kubernetes In Docker
+description: Example of deploying Istio locally using Kubernetes In Docker for testing and evaluation
+weight: 10
+keywords:
+- kubernetes
+- kind
+- metallb
+- cert-manager
+- ingress
+test: true
+---
+
+## About
+
+This example uses Kubernetes In Docker ([KIND][kind]) to deploy Istio locally. The cluster produced is intended to be a supplement for further testing, evaluation and experimentation with Istio.
+
+## Before you begin
+
+Make sure that you have [Docker][docker] and [Kind][kindquickstart] installed and that you are able to create and use kind clusters and connect to them with [Kubectl][kubectl].
+
+## Deploying Kubernetes
+
+For this example we'll be creating a default Kubernetes cluster using [Kind][kind] with a single node:
+
+{{< text bash >}}
+$ kind create cluster --name istio-testing
+{{< /text >}}
+
+{{< tip >}}
+If you're interested in building a multi-node cluster, or otherwise further customizing the cluster setup (e.g. different kubernetes versions, more worker nodes, e.t.c.) then be sure to check the [Kind Cluster Configuration Documentation][kindconfig] which will help you explore thea available options.
+{{< /tip >}}
+
+Once the cluster has completed deployment switch your [kubectl][kubectl] context to start using it:
+
+{{< text bash >}}
+kubectl cluster-info --context kind-istio-testing
+{{< /text >}}
+
+And then test access to the cluster by retrieving resources to make sure everyting is working (e.g. `kubectl get all -A`).
+
+## Enabling External LoadBalancers
+
+By default Istio will create a [LoadBalancer Service][servicelb] which will be resolved with an external address. For the purposes of this example we'll use [MetalLB][metallb] to resolve these to an address available on the docker network from the host machine.
+
+{{< tip >}}
+For this example we assume that the default docker network is used with an IP range of `172.17.255.1-172.17.255.250` (if you ran `kind` with no specifical configuration, this should be the default). If however you are using an alternative network configuration, you'll need to make the appropriate updates to the `addresses` field in the below MetalLB configuration before deploying.
+{{< /tip >}}
+
+First we'll pick the latest release and deploy the MetalLB components:
+
+{{< text bash >}}
+export LATEST_METALLB_RELEASE="$(curl -s https://api.github.com/repos/metallb/metallb/releases/latest | jq -r '.tag_name')"
+kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/${LATEST_METALLB_RELEASE}/manifests/namespace.yaml
+kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/${LATEST_METALLB_RELEASE}/manifests/metallb.yaml
+{{< /text >}}
```

Comment:
Every line this is a command (vs. output or extended multi-line command) must start with `$ `.
```suggestion
{{< text bash >}}
$ export LATEST_METALLB_RELEASE="$(curl -s https://api.github.com/repos/metallb/metallb/releases/latest | jq -r '.tag_name')"
$ kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/${LATEST_METALLB_RELEASE}/manifests/namespace.yaml
$ kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/${LATEST_METALLB_RELEASE}/manifests/metallb.yaml
{{< /text >}}
```

Reply:
:+1: